policy/data/query/jynx/query.go

```diff
@@ -0,0 +1,83 @@
+package jynx
+
+const (
+	ImagePackagesByDigestQueryName = "image-packages-by-digest"
+
+	ImagePackagesByDigestQuery = `
```

Comment:
I was wondering if we could type these queries too as we've already done in other projects such as the Scout CLI or the Prometheus exporter: https://github.com/atomisthq/scout-prometheus-exporter/blob/main/types/graphql_types.go#L10

Reply:
I think that we probably can't do that right now because of how async queries work - we'd need to change a load of interfaces and check if the GQL client is happy to spit out the actual query without trying to unmarshal the response as a GQL response. Worth looking into later though!